dev/com.ibm.ws.security.spnego/resources/com/ibm/ws/security/spnego/internal/resources/SpnegoMessages.nlsprops

```diff
@@ -113,3 +113,7 @@ SPNEGO_CAN_NOT_GET_DELEGATE_SERVICE_SPN.useraction=Verify the service principal
 SPNEGO_DELEGATE_SPN_LOGIN_TO_KDC_FAILURE=CWWKS4322E: Delegate SPN {0} can not authenticate to the Key Distribution Center (KDC) using the Kerberos configuration file {1} and keytab file {2}. Login exception was received: {3}
 SPNEGO_DELEGATE_SPN_LOGIN_TO_KDC_FAILURE.explanation=Can not authenticate to the Key Distribution Center (KDC).
 SPNEGO_DELEGATE_SPN_LOGIN_TO_KDC_FAILURE.useraction=Verify the service principal name format, and make sure it exists on the Windows domain and that the Kerberos keytab file contains the service principal name.
+
+SPNEGO_CONFLICTING_SETTINGS_CWWKS4323E=The attribute {0} from the {1} element conflicts with the attribute {2} from the {3} element. Specify only a value on either the {1} or the {3} element. It is suggested to specify only the value on the {1} element.
```

Comment:
suggested edits:
**change**
The attribute {0} from the {1} element conflicts with the attribute {2} from the {3} element. Specify only a value on either the {1} or the {3} element. It is suggested to specify only the value on the {1} element.
**to**
The {0} attribute from the {1} element conflicts with the {2} attribute  from the {3} element. Specify a value only on either the {1} or the {3} element, not on both elements. It is suggested to specify  the value only on the {1} element.

Do the attributes in this message need double bracketing?

**change**
Specify the attribute in only one of the config elements.
**to**
Specify the attribute in only one of the configuration elements.